bridge/src/commonMain/kotlin/ru/herobrine1st/matrix/bridge/telegram/TelegramWorker.kt

```diff
@@ -196,6 +200,60 @@ class TelegramWorker(
                                         RoomMessageEventContent.TextBased.Text(body = text),
                                     )
                                 }
+
+                                message.photo?.maxByOrNull { it.fileSize ?: 0 }?.let { photo ->
+
+                                    val sender = message.from!!
+
+                                    val fileInfo = withContext(Dispatchers.IO) {
+                                        bot.getFile(photo.fileId).toResult().get()
+                                    }
+
+                                    val caption = message.caption ?: "none"
```

Comment:
Leave it null, otherwise all messages will be with caption "none" on mx size
```suggestion
                                    val caption = message.caption
```